src/catalogue/items/catalogueItemsTable.component.tsx

```diff
@@ -271,35 +275,127 @@ const CatalogueItemsTable = (props: CatalogueItemsTableProps) => {
         accessorFn: (row) => row.item_model_number ?? '',
         size: 250,
       },
-
       {
         header: 'Manufacturer Name',
-        accessorFn: (row) => row.manufacturer.name,
-        size: 250,
-        filterVariant: 'autocomplete',
-        filterFn: 'equals',
-      },
-      {
-        header: 'Manufacturer Address',
-        accessorFn: (row) => row.manufacturer.address,
-        size: 350,
+        accessorFn: (row) =>
+          manufacturerList?.find((manufacturer) => {
+            return manufacturer.id === row.manufacturer_id;
+          })?.name,
+        Cell: ({ row }) => (
+          <MuiLink
+            underline="hover"
+            component={Link}
+            to={`/inventory-management-system/manufacturer/${row.original.manufacturer_id}`}
+          >
+            {
+              manufacturerList?.find((manufacturer) => {
+                return manufacturer.id === row.original.manufacturer_id;
+              })?.name
+            }
+          </MuiLink>
+        ),
       },
       {
         header: 'Manufacturer URL',
-        accessorFn: (row) => row.manufacturer.url,
-        size: 300,
+        accessorFn: (row) =>
+          manufacturerList?.find((manufacturer) => {
+            return manufacturer.id === row.manufacturer_id;
+          })?.url,
         Cell: ({ row }) => (
           <MuiLink
             underline="hover"
             target="_blank"
-            href={row.original.manufacturer.url}
+            href={
+              manufacturerList?.find((manufacturer) => {
+                return manufacturer.id === row.original.manufacturer_id;
+              })?.url ?? undefined
+            }
           >
-            {row.original.manufacturer.url}
+            {
+              manufacturerList?.find((manufacturer) => {
+                return manufacturer.id === row.original.manufacturer_id;
+              })?.url
+            }
           </MuiLink>
         ),
       },
+      {
+        header: 'Manufacturer Address',
+        accessorFn: (row) =>
+          `${
+            manufacturerList?.find((manufacturer) => {
+              return manufacturer.id === row.manufacturer_id;
```

Comment:
There are quite a few lines like this missing coverage - I think its because it doesn't wait for them to be loaded.